### ISSUE TITLE
freebsd completion: service does not work

### ISSUE THREAD

**Issue by rumpelsepp** [fish-shell/fish-shell#2715]
The autocompletion of `service` on freebsd does not work, since there is a wrong directory configured:

```
stefan@tatooine ~> sudo service myls: /etc/init.d: No such file or directory
```

The correct directories are: `/etc/rc.d` and `/usr/local/etc/rc.d`.

**Comment by faho** [comment]
Fixed by 57fcbec. I opted to use `service -l` since it seems that the local service directory at least might be configurable. Thanks!

**Comment by rumpelsepp** [comment]
:+1: Thanks!
